src/app/shared/models/Label.ts

```diff
@@ -0,0 +1,4 @@
+export class Label {
```

Comment:
That was actually generated by https://www.npmjs.com/package/angular2-swagger-client-generator or one of its forks using the swagger.json  .  However, there's still quite a few bugs in the generated code, so I omitted the process from .travis.yml